base/dates/conversions.jl

```diff
@@ -19,9 +19,20 @@ function unix2datetime(x)
 end
 # Returns unix seconds since 1970-01-01T00:00:00
 datetime2unix(dt::DateTime) = (value(dt) - UNIXEPOCH)/1000.0
+
+_tzoffset = 0
+function settzoffset()
+    t = floor(time())
+    tm = Libc.TmStruct(t)
+    dtutc = unix2datetime(t)
+    dtlocal = DateTime(tm.year+1900,tm.month+1,tm.mday,tm.hour,tm.min,tm.sec)
+    _tzoffset = Second(dtlocal - dtutc).value
+end
+settzoffset() #set it on load
+tzoffset() = _tzoffset
```

Comment:
The preferred way for a type stable mutable global is a const global of mutable type. You should design it such that you know what type this global is going to be.